cfme/networks/views.py

```diff
@@ -518,6 +519,20 @@ class NetworkRouterAddInterfaceView(BaseLoggedInPage):
     is_displayed = displayed_not_implemented
 
 
+class NetworkRouterAddCloudSubnetView(BaseLoggedInPage):
+    """ Represents Add Interface to Network Router page """
+    modal_title = Text('//div[@class="modal-header"]//h4')
+    subnet_name = TextInput(id='name')
+    address = TextInput(id='address')
+    netmask = TextInput(id='netmask')
+    gateway = TextInput(id='gateway')
+    add = Button('Add')
+
+    @property
+    def is_displayed(self):
+        return self.modal_title.text == 'Create L3 Cloud Subnet'
```

Comment:
There is no check here that this is also the add page for NetworkRouter CloudSubnet, and not just the basic CloudSubnet view.  Note that `class SubnetAddView` in this same module does not implement is_displayed - and I don't think this view should either.  Navigation to add CloudSubnet to a specific networkRouter needs to validate (or always navigate) that its adding the subnet to the correct NetworkRouter - and we aren't on an `Add` page from navigating through a different NetworkRouter.

Reply:
Just saw the old comments on this, and @digitronik suggestion to implement a method. I commented there as well, but figured I would complete the loop on this newer commit.

Apologies for the back and forth @aljazkosir, but I think you were right in the first place to not implement is_displayed.  It means this view always navigates, but that is better than incorrectly identifying a view and adding the subnet to the wrong NetworkRouter entity.